custom_model_runner/datarobot_drum/drum/artifact_predictors/torch_predictor.py

```diff
@@ -66,7 +66,11 @@ def predict(self, data, model, **kwargs):
         # done in the base class
         super(PyTorchPredictor, self).predict(data, model, **kwargs)
 
-        data = Variable(torch.from_numpy(data.values).type(torch.FloatTensor))
+        data = Variable(
```

Comment:
This isn't a deal breaker to include, but is probably nice to have if we want to allow users flexibility in what format they pass their data in after preprocessing. Currently we're assuming it will reach pytorch as a dataframe, but if tools like sklearn are used as part of the preprocessing flow this is not guaranteed to be the case.